src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -122,176 +108,84 @@ testRule(rule, {
   }, {
     code: "a { FONT-FAMILY: \"Lucida Grande\", Arial, sans-serif; }",
-    message: messages.expected("no", "Lucida Grande"),
-    line: 1,
-    column: 19,
-  }, {
-    code: "a { font-family: Lucida Grande, Arial, \"sans-serif\"; }",
-    message: messages.expected("no", "sans-serif"),
-    line: 1,
-    column: 41,
-  }, {
-    code: "a { font-family: Red/Black, Arial, sans-serif; }",
-    message: messages.expected("double", "Red/Black"),
-  }, {
-    code: "a { font-family: 'Red/Black', Arial, sans-serif; }",
-    message: messages.expected("double", "Red/Black"),
-  }, {
-    code: "a { font-family: Arial, Ahem!, sans-serif; }",
-    message: messages.expected("double", "Ahem!"),
   }, {
-    code: "a { font-family: Arial, 'Ahem!', sans-serif; }",
-    message: messages.expected("double", "Ahem!"),
-  }, {
-    code: "a { font-family: Hawaii 5-0, Arial, sans-serif; }",
-    message: messages.expected("double", "Hawaii 5-0"),
-  }, {
-    code: "a { font-family: 'Hawaii 5-0', Arial, sans-serif; }",
-    message: messages.expected("double", "Hawaii 5-0"),
-  } ],
-})
-
-testRule(rule, {
-  ruleName,
-  config: ["single-where-recommended"],
-
-  accept: [ {
-    code: "a { font-family: $sassy-font-family; }",
-    description: "ignores sass variables",
-  }, {
-    code: "a { font-family: @less-666; }",
-    description: "ignores less variables",
-  }, {
-    code: "a { font-family: var(--ff1); }",
-    description: "ignores custom properties",
-  }, {
-    code: "$font-family: map-get($font-stacks, default);",
-    description: "ignores scss variable contain font-family in name",
+    code: "a { font-family: Times, \"Times New Roman\", serif; }",
   }, {
-    code: "@font-family: map-get($font-stacks, default);",
-    description: "ignores less variable contain font-family in name",
+    code: "a { font-family: \"Something6\"; }",
   }, {
-    code: "$default-font-family: map-get($font-stacks, default);",
-    description: "ignores scss variable contain font-family in name",
+    code: "a { font-family: \"snake_case\"; }",
   }, {
-    code: "@default-font-family: map-get($font-stacks, default);",
-    description: "ignores less variable contain font-family in name",
+    code: "a { font-family: \"Red/Black\", Arial, sans-serif; }",
   }, {
-    code: "$font-family-variable: map-get($font-stacks, default);",
-    description: "ignores scss variable contain font-family in name",
+    code: "a { font-family: Arial, \"Ahem!\", sans-serif; }",
   }, {
-    code: "@font-family-variable: map-get($font-stacks, default);",
-    description: "ignores less variable contain font-family in name",
+    code: "a { font-family: \"Hawaii 5-0\", Arial, sans-serif; }",
   }, {
-    code: "a { font-family: 'Lucida Grande', Arial, sans-serif; }",
+    code: "a { font-family: 'Red/Black', Arial, sans-serif; }",
   }, {
-    code: "a { fOnT-fAmIlY: 'Lucida Grande', Arial, sans-serif; }",
+    code: "a { font-family: Arial, 'Ahem!', sans-serif; }",
```

Comment:
Lines 113 - 115, 148 - 153, 220 - 222 etc...

As with https://github.com/stylelint/stylelint/pull/1354, I merged the `"double-*-*"` and `"single-*-*"` options into their corresponding `"always-*-*"` options. The diff looks super confusing though :( Are you able to gander at source file instead?